data/korath/korath weapons.txt

```diff
@@ -152,6 +236,82 @@ effect "fire-lance impact"
 	"random angle" 360
 	"velocity scale" 0.
 
+outfit "Blaze-Pike Turret"
+	category "Turrets"
+	cost 1970000
+	thumbnail "outfit/blaze-pike"
+	"mass" 57
+	"outfit space" -54
+	"weapon capacity" -54
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/blaze-pike"
+			"frame rate" 20
+		"hardpoint sprite" "hardpoint/blaze-pike"
+		sound "blaze-pike"
+		"hit effect" "blaze-pike hit"
+		"inaccuracy" 0.45
+		"turret turn" 2.8
+		"velocity" 450
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 5
+		"firing heat" 4.5
+		"shield damage" 7.1
+		"hull damage" 3.1
+		"heat damage" 18
+		"burn damage" 0.45
+	description "The seeming reverence the Korath have for fire appears embodied in this weapon. This pair of Fire-Lances mounted together in a turret do not merely create twin lances of fire: Rather, they begin to form a gestalt -- a single, longer, hotter, and almost hypnotic column of fire."
+
+effect "blaze-pike hit"
+	sprite "effect/blaze-pike hit"
+		"no repeat"
+		"frame rate" 30
+	"lifetime" 5
+	"random angle" 360
+	"random spin" 1
+	"random velocity" 5
+	"velocity scale" .005
+
+outfit "Korath Inferno"
+	category "Turrets"
+	cost 2670000
+	thumbnail "outfit/korath inferno"
+	"mass" 77
+	"outfit space" -77
+	"weapon capacity" -77
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/korath inferno"
+			"frame rate" 20
+		"hardpoint sprite" "hardpoint/korath inferno"
+		sound "korath inferno"
+		"hit effect" "korath inferno hit"
+		"inaccuracy" 0.5
+		"turret turn" 2
+		"velocity" 550
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 7.5
+		"firing heat" 8.5
+		"shield damage" 10.8
+		"hull damage" 4.8
+		"heat damage" 30
+		"burn damage" 0.9
+	description "Some Korath consider the Inferno the crowning achievement of Exile weapon design. The three Fire-Lances mounted together in precisely the right manner combine synergistically, creating a longer, more powerful beam than three would individually."
```

Comment:
```suggestion
	description "Some Korath consider the Inferno the crowning achievement of Exile weapon design. The three Fire-Lances mounted together in precisely the right manner combine synergistically, creating a longer, more powerful beam than spread out individually."
```

Reply:
This doesn't sound correct at all?

Are you perhaps confusing than with then?

Reply:
I'm not quite feeling that proposed change, I think it misses the intent a little. For an alternative that I think sounds a better:

```suggestion
	description "Some Korath consider the Inferno the crowning achievement of Exile weapon design. The three Fire-Lances mounted together in precisely the right manner combine synergistically, creating a longer, more powerful beam than each would individually."
```

Reply:
I feel a change here to this description is unnecessary, it already makes sense. But if we are doing your suggestion z, it should be "than each would *be* individually l"

Reply:
Each implies separation to me, three drives home that this is more than the sum of its parts.

Reply:
Just about there, would `A trio of Fire-Lances` work instead?

Reply:
I still hate it but it's better than triplet